wp-content/themes/currentorg/less/wpbdp.less

```diff
@@ -444,6 +444,26 @@
 }
 
 // submit a listing page
+.wpbdp-submit-listing-flex-grid {
+  display: flex;
+  .wpbdp-fee-plan-img {
+    flex: 1;
+    text-align: center;
+    img {
+      max-width: 95%;
+      max-height: 515px;
+      @media (max-width:1000px) {
+        max-height: 100%;
+      }
+    }
+    @media (max-width:1000px) {
+      margin-bottom: 10px;
+    }
+  }
+  @media (max-width:1000px) {
```

Comment:
1000px seems kind of wide for this:

![Screen Shot 2019-07-08 at 16 44 55 ](https://user-images.githubusercontent.com/1754187/60841425-cba57100-a19f-11e9-9a8d-80c5f97617ba.png)